src/fava/beans/funcs.py

```diff
@@ -6,16 +6,21 @@
 from typing import TYPE_CHECKING
 
 from beancount.core import compare  # type: ignore[attr-defined]
-from beancount.query import query  # type: ignore[attr-defined]
-from beancount.query import query_execute  # type: ignore[attr-defined]
+from beanquery import query  # type: ignore[import-untyped]
+from beanquery import query_execute
```

Comment:
`beanquery.query_execute` is deprecated and kept only for a minimal form of backward compatibility.

Reply:
Indedd, I thought this was `beanquery.query.run_query`. Looking into `beanquery.query_execute` is most likely looking for unnecessary trouble.

Reply:
I don't like the API of `beanquery.numberify.numberify_results()` and I would like to replace it with something else, but I haven't thought about it much yet, so it will not go away anytime soon. You can apply `beanquery.numberify.numberify_results()` to the return value of `beanquery.Connection.execute()`:
```python
    conn = beanquery.connect('beancount:', ...)
    curs = conn.execute(query.format(*args))
    rrows = curs.fetchall()
    rtypes = curs.description
    dformat = options['dcontext'].build()
    rtypes, rrows = beanquery.numberify.numberify_results(rtypes, rrows, dformat)
```
it is nor pretty but it works and you will avoid a `DeprecationWarning` that I'll be adding to `beanquery.query.run_query()` soon.

Reply:
cool, changed as you suggested :)